src/components/markdown-renderer/markdown-extension/katex/katex-frame.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useMemo } from 'react'
+import KaTeX from 'katex'
+import convertHtmlToReact from '@hedgedoc/html-to-react'
+import 'katex/dist/katex.min.css'
+import { Alert } from 'react-bootstrap'
+import { testId } from '../../../../utils/test-id'
+import { sanitize } from 'dompurify'
+
+interface KatexFrameProps {
+  expression: string
+  block?: boolean
+}
+
+/**
+ * Renders a LaTeX expression.
+ *
+ * @param expression The mathematical expression to render
+ * @param block Defines if the output should be a block or inline.
+ */
+export const KatexFrame: React.FC<KatexFrameProps> = ({ expression, block }) => {
```

Comment:
Should the default of `block` not be explicitly set instead of being worked around with the `?`?